app/controllers/api/v1/projects_controller.rb

```diff
@@ -22,25 +30,25 @@ def create
     render json: { project: project, price: price, access: project_access }, status: :ok
   end
 
-  # TODO update partially project
   def update
-    project = Project.find(params[:projectId])
-    puts "Vamo' a actualizar #{project.name}"
+    @project.attributes = project_params[:project]
   end
 
   def list
-    projects = Project.all
-    render json: projects, status: :ok
+    projects = Project.all.order(:created_at)
+    if search_params[:type]
+      projects = Project.where("type_project = ?", search_params[:type]).order(:created_at)
```

Comment:
puedes usar el projects q ya ordenaste antes
```suggestion
      projects = projects.where("type_project = ?", search_params[:type]).order(:created_at)
```